docs/userguide/labels-custom-metadata.md

```diff
@@ -1,230 +1,119 @@
 
-Another user may apply the same label based on a building's "architecture":
+- Authors of third-party tools should prefix each label key with the
+  reverse DNS notation of a domain they own, such as `com.example.some-label`.
 
-    LABEL architecture="Art Nouveau"
-
-To prevent naming conflicts, Docker recommends using namespaces to label keys
-using reverse domain notation. Use the following guidelines to name your keys:
-
-- All (third-party) tools should prefix their keys with the
-  reverse DNS notation of a domain controlled by the author. For
-  example, `com.example.some-label`.
+- Do not use a domain in your label key without the domain owner's permission.
 
 - The `com.docker.*`, `io.docker.*` and `org.dockerproject.*` namespaces are
-  reserved for Docker's internal use.
-
-- Keys should only consist of lower-cased alphanumeric characters,
-  dots and dashes (for example, `[a-z0-9-.]`).
-
-- Keys should start *and* end with an alpha numeric character.
-
-- Keys may not contain consecutive dots or dashes.
-
-- Keys *without* namespace (dots) are reserved for CLI use. This allows end-
-  users to add metadata to their containers and images without having to type
-  cumbersome namespaces on the command-line.
-
-
-These are simply guidelines and Docker does not *enforce* them. However, for
-the benefit of the community, you *should* use namespaces for your label keys.
-
-
-## Store structured data in labels
-
-Label values can contain any data type as long as it can be represented as a
-string. For example, consider this JSON document:
-
-
-    {
-        "Description": "A containerized foobar",
-        "Usage": "docker run --rm example/foobar [args]",
-        "License": "GPL",
-        "Version": "0.0.1-beta",
-        "aBoolean": true,
-        "aNumber" : 0.01234,
-        "aNestedArray": ["a", "b", "c"]
-    }
-
-You can store this struct in a label by serializing it to a string first:
-
-    LABEL com.example.image-specs="{\"Description\":\"A containerized foobar\",\"Usage\":\"docker run --rm example\\/foobar [args]\",\"License\":\"GPL\",\"Version\":\"0.0.1-beta\",\"aBoolean\":true,\"aNumber\":0.01234,\"aNestedArray\":[\"a\",\"b\",\"c\"]}"
-
-While it is *possible* to store structured data in label values, Docker treats
-this data as a 'regular' string. This means that Docker doesn't offer ways to
-query (filter) based on nested properties. If your tool needs to filter on
-nested properties, the tool itself needs to implement this functionality.
-
-
-## Add labels to images
-
-To add labels to an image, use the `LABEL` instruction in your Dockerfile:
-
-
-    LABEL [<namespace>.]<key>=<value> ...
-
-The `LABEL` instruction adds a label to your image. A `LABEL` consists of a `<key>`
-and a `<value>`.
-Use an empty string for labels  that don't have a `<value>`,
-Use surrounding quotes or backslashes for labels that contain
-white space characters in the `<value>`:
-
-    LABEL vendor=ACME\ Incorporated
```

Comment:
When I first saw all this being removed I thought "what?" but it's all duplicated in other parts of the docs! Nice cleanup 💯